packages/openssl.rb

```diff
@@ -1,12 +1,64 @@
 require 'package'
 
 class Openssl < Package
-  description 'OpenSSL is a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols.'
-  homepage 'https://www.openssl.org/'
-  version '3.0.2'
+  description 'The Open Source toolkit for Secure Sockets Layer and Transport Layer Security'
+  homepage 'https://www.openssl.org'
+  _ver = '1.1.1i'
+  version "#{_ver}"
   compatibility 'all'
+  source_url "https://www.openssl.org/source/openssl-#{_ver}.tar.gz"
```

Comment:
I'm thinking it's not worth it to create a `_ver` variable if it's only used twice?

Reply:
Why not using the version variable directly? (`source_url "https://www.openssl.org/source/openssl-#{version}.tar.gz"`)

Reply:
So this is a hack that arch is using, which could work for us too.

```_ver ``` is source package version
crew package```version``` is ```#{_ver}-1``` or ```#{_ver}-2``` or whatever.
If package sources have the source in the URL, this makes updates easier, no?

Reply:
Say we make a modification to the package file, e.g. hey clang makes this library only 3 bytes! or whatever.
In that case source_url breaks since we are incrementing ```version```.

Reply:
I'm not wedded to this approach. Just trying to make incremental improvements for maintainability.

Reply:
That is a solid case for doing it this way.  I also think you could simply use `version _ver` and it would work the same (no need to put in quotes).  If we did that, then you could do something like `version _ver + '-1'`.

Reply:
Modified accordingly.